src/sentry/api/endpoints/organization_recent_searches.py

```diff
@@ -70,5 +70,5 @@ def post(self, request, organization):
                 remove_excess_recent_searches(organization, request.user, result["type"])
             status = 201 if created else 204
 
-            return Response("", status=status)
```

Comment:
I guess, what I jsut realized made this worse and less obvious, this isn't an empty string according to DRF, this ends up getting JSON encoded into the literal `""` instead of no body.

If you do a normal `return HttpResponse("", status=204)`, things are fine. But going through DRF is effectively the same as `return HttpResponse(json.dumps(""), status=204)` which makes a body of a content length of 2.